lib/binary_to_decimal.rb

```diff
@@ -4,6 +4,51 @@
 # The least significant bit is at index 7.
 # Calculate  and return the decimal value for this binary number using
 # the algorithm you devised in class.
+
+require 'pry'
```

Comment:
Not really a problem here, but when you're finished debugging etc and turn code into production, remove pry from the codebase.